lua/lvim/plugins.lua

```diff
@@ -10,19 +10,19 @@ local commit = {
   fixcursorhold = "0e4e22d21975da60b0fd2d302285b3b603f9f71e",
   friendly_snippets = "d4f5c0507cfe4c67024f58c84ba982f7f5c71a7a",
   gitsigns = "f4648dcc055a10573f36e83343592824155ab466",
-  lua_dev = "03a44ec6a54b0a025a633978e8541584a02e46d9",
+  lua_dev = "a0ee77789d9948adce64d98700cc90cecaef88d5",
   lualine = "70691ae350fdbe1f15758e3b8e2973742a7967a9",
   luasnip = "0222ee63c9e4b80e6000d064f8efd8edcc6d0c48",
   nlsp_settings = "3a3942b5d1da30e3ca0dc431aada3191c5952054",
-  null_ls = "afd9a999fa29077e12b7284cf11dff37972fbce5",
+  null_ls = "f9b31c07a2abe97860cd58a06127f7d4e8ee6582",
   nvim_autopairs = "97e454ce9b1371373105716d196c1017394bc947",
-  nvim_cmp = "4c0a6512a0f8a235213959badf70031b9fa0220a",
+  nvim_cmp = "8fc8d31cf451b8becb39087aa2190bee29a8b009",
   nvim_dap = "08349ab5da2148965deb2e778065017059bff584",
-  nvim_lsp_installer = "9894db12783fdd02be909cdd60e6d0fc20b9e46e",
+  nvim_lsp_installer = "524553022ca5ad29608dab9568012076aa17ee0d",
   nvim_lspconfig = "09130804eb27fd31539922967d14c28388765ae4",
   nvim_notify = "15f52efacd169ea26b0f4070451d3ea53f98cd5a",
   nvim_tree = "0a2f6b0b6ba558a88c77a6b262af647760e6eca8",
-  nvim_treesitter = "723d91e8217ae66ea75f809f404d801ed939f497",
+  nvim_treesitter = "8b1d3906199e1572c2201e494bba017d823c0c8f",
```

Comment:
btw this causes weird issues for neovim head. Can we keep nvim_treesitter for another PR to make sure it doesn't break anything